acme/messages.py

```diff
@@ -182,15 +182,27 @@ def emails(self):
 
     @property
     def phone(self):
-        """Phone."""
-        assert len(self.phones) == 1
-        return self.phones[0]
+        """Phone.
```

Comment:
There are many other places that rely on just singular email and it doesn't make sense to fix it now, as my next PR just scrambles everything. I will make sure to remove singulars in next PR.